en/docs/apis/api-concepts.md

```diff
@@ -1,14 +1,14 @@
 # API Concepts
-Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
+Choreo API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
```

Comment:
Seems like this page also has[Choreo Console]({{choreo_console}}) leading to a broken link.

Reply:
Fixed. Thanks